data/ocicrypt_config.json

```diff
@@ -0,0 +1,13 @@
+{
+  "key-providers": {
+    "keyprovider": {
+      "cmd": {
+        "path": "/usr/lib/keyprovider-wrapkey",
+        "args": []
+      }
+    }
+  },
+  "keyprovider2": {
```

Comment:
I think this needs to be in the above `key-providers` field? for consistency, we can rename the above to `keyprovider1` as well?